chart/kubeapps/values.yaml

```diff
@@ -66,6 +66,7 @@ frontend:
   service:
     port: 80
     type: ClusterIP
+    loadBalancerIP:
```

Comment:
I prefer not having `null` values here (parameters without a value). I would either remove this line or comment it (adding an additional comment with its meaning).

Reply:
Thanks, I've added a short description and commented out the parameter.